### PR TITLE
New technology: biomass gasification with carbon capture

### PR DESCRIPTION
**Pull request by robinhasse**
This PR adds a new PE2SE technology `biogasc` that represents biomass gasification with carbon capture. Test runs also with high mitigation did not show any deployment of this technology probably due to the uncompetitive capture rate.

# Parameterisation

The technology is parameterised consistent with convetional biomass gasification `biogas`. The table below compares both technologies. The last column points to explanation given below.

## Overview

parameter | unit | biogas | biogasc | explanation
-- | -- | -- | -- | --
CO2 emissions |  $Gt_c/ZJ_{PE}$ | 0 |  -9.5 |  1.
captured CO2 | $Gt_c/ZJ_{PE}$  | 0 | 9.5 |  1.
N20 emissions | $Tg_N/TWa$  | 0.12614 | 0.12614 | 1.
technology status |   | 1 | 3 |   
capacity factor |   | 0.91 | 0.91 |  3.
efficiency |   | 0.55 | 0.55 |  2.
electricity cogeneration |   | 0 | -0.07 |  2.
construction time | $a$ | 4 | 4 |   
lifetime | $a$ | 40 | 40 | 4.
vintage |   | 50% in 2005    33% in 2000    17% in 1995 | 100% in 2005 |   
mix0 |   |   |   |  6.
initial investment cost | $USD_{2015}/kW_{SE}$  | 1200 | 1230 |  5., 6.
O&M cost fixed | $1/a$ | 0.05 | 0.06 |  5.
O&M cost variable |  $USD_{2015}/kWa$ | 10.90 | 12.8 |  5.

### 1. Emissions
- Koornneef et al. simulate i.a. the gasification of woody biomass with methanation and CO2 capture [1], [2]. They assume a CO2 removal efficiency of $0.36$ in 2030 and $0.38$ in 2050 with biomethane as product [1, p. 6047]. A carbon content of $100~ kg_{CO2}⁄GJ  = 27.3 ~Gt_C⁄ZJ$ is assumed which inexplicitly includes indirect GHG emissions from the electricity consumption for the CO2 compression [1, p. 6048]. The carbon removal efficiency for REMIND (where processing captured CO2 is processed separately) would thus be slightly higher.
- Carbon removal efficiency is assumed to be $0.38$ resulting in negative CO2 emissions and captured CO2 of $9.5~Gt_C/ZJ_{PE}$ .

### 2. Efficiency
- We propose to define biogas and `biogasc` with identical efficiency to assure consistency. Technically, both processes are mostly identical except from the capture of CO2 which requires additional electricity. $55~\%$ seems to be at the lower end of the spectrum of expected efficiencies so it is worth considering to update it to $60~\%$ or even $65~\%$. The latter does not exceed anticipated future values in the literature [1], [11] but is higher than simulation results that refer to currently existent technology [4], [7]. Based on [4], [8], we suggest an electricity own-consumption of $0.07$ (relative to biomethane generation) and to neglect other co-generation such as heat.

### 3. Capacity factor
- Koornneef et al. assume a capacity factor of $0.91$ [2, p. 51] which is identical to `biogas` in REMIND, Michailos et al. assume $0.9$ [4].

### 4. Lifetime
- Assumed values in literature that is included here are $15~ years$ [3], $20~ years$ [4], [10] and $30~ years$ [2], [5].
- To be consistent, we propose to use the same lifetime as for `biogas`. $40 ~ years$ seems to be within the wide range of uncertainty [2, p. 25].

### 5. Cost
- We propose to take `biogas` investment cost as a reference and add $30 ~USD_{2015}/kW$ for the additional CO2 capture inline with [1]. The additional O&M cost from [1] are separated into fixed and variable share maintaining the ratio seen for `biogas` in REMIND and are added to the respective O&M cost of `biogas`. This results in fixed O&M of $0.057$ (relative to investment) and $12.77 ~USD_{2015}/kWa$.

### 6. Alternative scenarios
- There are alternative technical parameters for SSP1 and SSP2, e.g. investment cost for some fossil technologies change and some technologies have set mix0 (i.a. `biogas`). We have not yet parameterised `biogasc` differently across SSPs.

## References
[1]	J. Koornneef, P. van Breevoort, P. Noothout, C. Hendriks, L. Luning, and A. Camps, ‘Global Potential for Biomethane Production with Carbon Capture, Transport and Storage up to 2050’, Energy Procedia, vol. 37, pp. 6043–6052, 2013, doi: 10.1016/j.egypro.2013.06.533.
[2]	Koornneef, Joris, P. van Breevoort, P. Noothout, C. Hendriks, and L. Luning, ‘Potential for biomethane production with carbon dioxide capture and storage’. International Energy Agency, 2013, Accessed: Feb. 24, 2021. [Online]. Available: https://ieaghg.org/docs/General_Docs/Reports/2013-11.pdf.
[3]	M. C. Carbo, R. Smit, B. van der Drift, and D. Jansen, ‘Bio energy with CCS (BECCS): Large potential for BioSNG at low CO2 avoidance cost’, Energy Procedia, vol. 4, pp. 2950–2954, 2011, doi: 10.1016/j.egypro.2011.02.203.
[4]	S. Michailos, O. Emenike, D. Ingham, K. J. Hughes, and M. Pourkashanian, ‘Methane production via syngas fermentation within the bio-CCS concept: A techno-economic assessment’, Biochem. Eng. J., vol. 150, p. 107290, Oct. 2019, doi: 10.1016/j.bej.2019.107290.
[5]	M. Chandel and E. Williams, ‘Synthetic Natural Gas (SNG): Technology, Environmental Implications, and Economics’. Energy Research Centre of the Netherlands, 2009.
[6]	B. Erlach, B. Harder, and G. Tsatsaronis, ‘Combined hydrothermal carbonization and gasification of biomass with carbon capture’, Energy, vol. 45, no. 1, pp. 329–338, Sep. 2012, doi: 10.1016/j.energy.2012.01.057.
[7]	C. M. van der Meijden, H. J. Veringa, and L. P. L. M. Rabou, ‘The production of synthetic natural gas (SNG): A comparison of three wood gasification systems for energy balance and overall efficiency’, Biomass Bioenergy, vol. 34, no. 3, pp. 302–311, Mar. 2010, doi: 10.1016/j.biombioe.2009.11.001.
[8]	Marija Saric, J. W. Dijkstra, L. Rabou, and Stéphane Walspurger, ‘Power-to-gas coupling to biomethane production’. Energy Research Centre of the Netherlands, 2013, Accessed: Feb. 24, 2021. [Online].
[9]	Marija Saric, J. W. Dijkstra, and S. Walspurger, ‘Power-to-Gas coupling to biomethane production: a feasibility study’, presented at the International Conference on Polygeneration Strategies, Vienna, 2013, Accessed: Jan. 03, 2021. [Online]. Available: https://www.biosng.com/fileadmin/biosng/user/documents/reports/Power-to-Gas_coupling_to_biomethane_production_-_a_feasibility_study.pdf.
[10]	A. Skorek-Osikowska, M. Martín-Gamboa, and J. Dufour, ‘Thermodynamic, economic and environmental assessment of renewable natural gas production systems’, Energy Convers. Manag. X, vol. 7, p. 100046, Sep. 2020, doi: 10.1016/j.ecmx.2020.100046.
[11]	S. Fendt, A. Buttler, M. Gaderer, and H. Spliethoff, ‘Comparison of synthetic natural gas production pathways for the storage of renewable energy’, WIREs Energy Environement, vol. 5, no. 3, pp. 327–350, 2016, doi: 10.1002/wene.189.
[12]	G. Aranda, A. van der Drift, and R. Smit, ‘The Economy of Large Scale Biomass to Substitute Natural Gas (bioSNG) plants’. Energy Research Centre of the Netherlands, 2014, Accessed: Jan. 03, 2021. [Online]. Available: https://www.biosng.com/fileadmin/biosng/user/documents/reports/The_Economy_of_Large_Scale_Biomass_to_Bio-Methane.pdf.

### CODE REVIEW

**Comment by robinhasse** [comment]
@robertpietzcker Is there anyone else who should take a look at this?

**Comment by robertpietzcker** [comment]
> @robertpietzcker Is there anyone else who should take a look at this?

I would say Nico.

> * we suggest an electricity own-consumption of 0.07

in the table the field is empty :-)

**Comment by nicobauer** [comment]
Hi Robin, sorry for the late reply and thanks for the reminder once more.

Regarding the uncompetitiveness I would expect that we may obeserve here typical issue of a new technology that the optimization algorithm does not bring into the system due to numerical issues. Thus it is an artefact.

There is a standard test. I do not know whether you did that already. It goes like this:

Abolish all other BECCS techs and rerun the model in a tight budget mode with some lower bound on the investment of biogasc. That should make the tech competitive. Then remove both bounds again and rerun the model from the gdx of the previous run. If biogasc is reduced and appears in a more balanced way in the investment portfolio we should expect it to be compettive. If it disapperars again, then we should consider that tech-option a true no-burner.

**Comment by robinhasse** [comment]
Thanks Nico for taking a look and your test advice!

> There is a standard test. I do not know whether you did that already.

I have not performed the test you described but ran a test run with a unrealistically high capture rate which lead to deployment of the new technology. That at least gave some confidence, that the implementation is technically correct. Of course, I can also perform your test. But I don't quite understand the reasoning. We always use baseline runs as starting point gdx. We would not find any BECCS deployment there, right? But this does not prevent the deployment of already implemented technologies in mitigation scenarios. I don't understand how this should be relevant for new technologies then.

File: core/input/generisdata_emi.prn [hunk 1]
```diff
@@ -35,8 +35,9 @@ pecoal.seliqfos.coalftrec      26.1
 pecoal.seliqfos.coalftcrec      7.8      18.3
 pecoal.sesofos.coaltr          26.1             0.04415
 pecoal.segafos.coalgas         26.1             0.04415
-pebiolc.seel.bioigccc         -20.0      20.0   
-pebiolc.seel.bioigcc                            
+pebiolc.seel.bioigccc         -20.0      20.0
+pebiolc.seel.bioigcc
+pebiolc.segabio.biogasc        -9.5       9.5   0.12614
```
Comment (merfort) [inline core/input/generisdata_emi.prn:40]: With [this bugfix](https://github.com/remindmodel/remind/pull/885) we removed the N2O emission factors from generisdata_emi.prn for pebiolc technologies completely.
```suggestion
pebiolc.segabio.biogasc        -9.5       9.5
```

**Review by LaviniaBaumstark — APPROVED** [review]

**Pull request merged**